website/sidebars.json

```diff
@@ -54,7 +54,8 @@
       "development/front-end/json-rpc"
     ],
     "Ecosystem Tools": [
-      "development/tools/subkey"
+      "development/tools/subkey",
+      "development/tools/key-management"
```

Comment:
I don't know if I'd call key management a tool. It should go in a different section.